### PR TITLE
Feat/32 new template with html

### LINKED ISSUE #32: [FEAT] Email Templates > SuperAdmin Email Management Dashboard > New Template with HTML
## Description
Create the Page for Generating New Email Template with HTML found in the Super Admin Email Management Dashboard

## Acceptance Criteria
-  Reusable Breadcrumbs
- Page title and description
-  Well validated form with input and button submission
- Tips Component with a defined title and bulleted tips
- Components adheres to the brand colors and design
- Responsive design of page

## Expected Outcome

- User should be able to see a tips card for helpful information
- User should be able to input or paste link to a html file in the input field
- User should be able to click the button to generate the new html template
- On success, user should be directed to the preview generated html page

## Purpose
As a User, I want to input a link pointing to my Html file to generate a new email template

## Requirement

-  Installs and uses shadcn-ui alert
-  Makes use of the form, Input and button components
-  Makes use of the Breadcrumbs component

## Testing

- [ ] test for input validation
- [ ] test for link submission
- [ ] test for template generation

## Task
- [ ] Has Breadcrumbs for easy navigation
- [ ] Implement Page with a clear title and description
- [ ] Implement Tips component using shadcn-ui alert
- [ ] Implement Form input with label and button
- [ ] validate that link from form input points to .html file
- [ ] button generates template on form submission
- [ ] navigate to preview template page on successful template generation
- [ ] Components adhere to the brand design and color scheme
- [ ] Page is responsive across device widths

## Deadline
You are given till Saturday 20th July 10:00AM WAT to complete the task.

## Picture
generate_email_template

## Dev Mode Figma Link
https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=63-69677&m=dev

### PR DESCRIPTION
**Pull request by kingbary**
# [FEAT] Email Templates > SuperAdmin Email Management Dashboard > New Template with HTML

**Closes #32 **

# Changes proposed

## What were you told to do?
 # Create the Page for Generating New Email Template with HTML found in the Super Admin Email Management Dashboard

## What did you do?
Create breadcrumbs for easy navigation
Implement Page with a clear title and description
 Implement Tips component using shadcn-ui alert
 Implement Form input with label and button
 Validate the link from the form input points to the .html file
 Make button generates a template on form submission
 Navigate to the preview template page on successful template generation
 Components adhere to the brand design and colour scheme
 Make Page responsive across device widths

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos
### [Figma file Link](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=63-69677&m=dev)
generate-email-template-screenshot-1
generate-email-template-screenshot-2

### CODE REVIEW

**Comment by kingbary** [comment]
@chumex412 @Samadeen @shalomtaiwo  Please review the PR

File: .gitignore
Comment (prudentbird) [inline .gitignore:1]: Why are you changing git ignore??

File: .vscode/extensions.json
Comment (prudentbird) [inline .vscode/extensions.json:1]: why are you changing VSCode settings??

File: package.json
Comment (prudentbird) [inline package.json:1]: Why are you installing dependencies??

**Review by prudentbird — CHANGES_REQUESTED** [review]
Follow all guidelines

File: .vscode/extensions.json
Comment (kingbary) [reply .vscode/extensions.json:1]: Fixed!

**Review by kingbary — COMMENTED** [review]
I have updated it now.

**Comment by oreoluwa212** [comment]
Checks are failing, fix up!

**Comment by kingbary** [comment]
I couldn't run the test locally that's why the test failed. I have fixed the error I got the test console now. Please rerun the test

**Review by prudentbird — CHANGES_REQUESTED** [review]
Fix up all your errors, you changed config files, go back and remove all the changes !


All your tests are failing!

**Comment by kingbary** [comment]
I could run the test locally, I fixed all the test errors now. Please review!

**Comment by oreoluwa212** [comment]
`Conflicting files
.github/workflows/lint.yml
.github/workflows/staging-cicd.yml
.vscode/extensions.json
.vscode/settings.json
docker/staging/docker-compose.yml
package-lock.json
src/app/(admin)/layout.tsx
src/app/(home)/page.tsx
src/app/globals.css
src/app/guides/page.tsx
src/components/common/Breadcrumb/README.md
src/components/common/Breadcrumb/index.tsx
src/components/layouts/homepage/Hero.tsx
src/components/superadminlayout/navbar/AdminNavbar.tsx
src/components/ui/button.tsx
src/components/ui/dialog.tsx
tsconfig.json`

**Comment by oreoluwa212** [comment]
you have to move your folders around, after pulling and retrieve the components that have been done by other interns, we cant accept repetition
N.B you can check the github issues [here](https://github.com/hngprojects/hng_boilerplate_nextjs/issues) for easy location,

**Pull request closed without merging**